src/styles/Landing.module.css

```diff
@@ -16,6 +16,13 @@
   background: linear-gradient(225deg, #005eaa 0%, #26418f 100%);
 }
 
+.build-number {
+  position: absolute;
+  bottom: 0;
+  color: #cacaca;
```

Comment:
Is there a variable that we could use or is this the most appropriate hex value?

Reply:
Ideally, we would use the theme variables provided by cdc-react.  We'll need to work on that integration in another ticket.